R/plumber.R

```diff
@@ -196,7 +196,7 @@ plumber <- R6Class(
 
     },
     run = function(host='127.0.0.1', port=getOption('plumber.port'), swagger=interactive(),
-                   debug=interactive() ){
+                   debug=interactive(), swagger.url=getOption('plumber.swagger.url', NULL)){
```

Comment:
Do you mind renaming this to make it more clear that it's a callback? As written, I feel like it implies that you get to specify the URL using this argument.

And I've usually been using camelCase for parameter names.

Maybe this could be `swaggerCallback`?

Reply:
Sure, that'll work. I'll change it on Monday and update the PR.